src/wp-admin/edit.php

```diff
@@ -389,7 +389,7 @@
 	/* translators: %s: Number of patterns. */
 	'deleted'   => _n( '%s pattern permanently deleted.', '%s patterns permanently deleted.', $bulk_counts['deleted'] ),
 	/* translators: %s: Number of patterns. */
-	'trashed'   => _n( '%s pattern moved to the Trash.', '%s patterns moved to the Trash.', $bulk_counts['trashed'] ),
+	'trashed'   => _n( '%s pattern moved to the Trash.', '%s patterns moved to the Trash.', $bulk_counts['trashed'] ) . sprintf( __( 'It will be deleted from trash after %s days.' ), EMPTY_TRASH_DAYS ),
```

Comment:
```suggestion
'trashed'   => _n( '%s pattern moved to the Trash.', '%s patterns moved to the Trash.', $bulk_counts['trashed'] ) . _n( 'It will be deleted from the trash after %s day.', 'It will be deleted from the trash after %s days.', EMPTY_TRASH_DAYS ),
```